### DISCUSSION TITLE
HmIP-ESI Ausgabewert wird nach Firmware Update zurückgesetzt

### DISCUSSION THREAD

**Discussion by jaal2001** [OpenCCU/OpenCCU#2745] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Hallo zusammen,
ich habe das Firmware Upgrade des HmIP-ESI durchgeführt 1.4.0.
Die Zählerwerte, die in der CCU3 angezeigt werden, sind korrekt. 
Ich gebe aber zusätzlich den Wert des Zählers (homematic/status/Gaszaehler:2/GAS_VOLUME) per MQTT nach außen. Dieser Wert war bisher ansteigend, ist mit dem Firmware Upgrade aber auf 0 gesetzt worden.

Ggfs. bin ich mit der Meldung auch hier falsch, weil der Wert im Device durch das Upgrade gelöscht wurde. Dann gerne den Hinweis an mich, dann würde ich mal eQ-3 kontaktieren.

Lasst mich gerne wissen, welche Logs/Infos, etc. ich liefern soll.

### Describe the behavior you expected

Wert sollte bestehen bleiben.

### Steps to reproduce the issue

Unklar ob reproduzierbar.

### What is the version this bug report is based on?

3.75.7.20240420

### Which base platform are you running?

generic-aarch64 (Generic appliance, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

n/a

### Anything in the logs that might be useful for us?

```txt
n/a
```

**Comment by jens-maus** [comment, marked as answer]
Ich hab zwar selbst den HmIP-ESI nicht, kann mir aber vorstellen das es sich mit diesem Datenpunkt ähnlich verhält wie mit den Datenpunkten beim älteren HM-ES-TX-WM der selbst nach einem Batteriewechsel einige Datenpunkte dann eben auf 0 setzt. D.h. Das sind quasi Datenpunkte die nur während der dauerhaften Laufzeit des Gerätes ansteigend sind und dann es sogar einen Überlauf geben kann wenn das Limit des Datentypes dieses Datenpunktes irgendwann erreicht ist.

Es ist aber in jedem Fall kein Bug/Issue von RaspberryMatic selbst und gehört daher eher in den Diskussionebereich (Issue=Bug !).

**Reply by jaal2001** [reply]
Okay, danke. Mir war nicht ganz klar, wo das Problem ursächlich herkommt, daher hatte ich das erstmal als Bug eröffnet.
Gibt es eine einfache Möglichkeit die Datenpunkte wieder auf den korrekten Wert zu setzen? Oder geht nur per Scripting - da bin ich nicht ganz firm drin, aber üblicherweise gibt es ja immer jemanden, der im Internet die Lösung publiziert :-)

**Reply by Baxxy13** [reply]
Die Datenpunkte sind "read-only", also nein es ist auch nicht per Script möglich. 
Das was du in der WebUI siehst ist eine systeminterne Systemvariable die per systeminternen Programm samt Script gefüllt wird. Die solltest du aufspüren und deren Daten nutzen.

**Reply by jaal2001** [reply]
Danke!

**Reply by jaal2001** [reply]
@Baxxy13 Wenn man richtig anfängt zu suchen, dann findet man auch sehr gute fertige Skripte: https://homematic-forum.de/forum/viewtopic.php?f=31&t=82352 ;-)
Danke für deine Arbeit. Ohne dein Skript würde ich das nicht hinbekommen.
